src/app/obzboard-set.ts

```diff
@@ -106,6 +108,53 @@ export class OBZBoardSet implements ImageResolver, SoundResolver {
     return observables.length > 0 ? forkJoin(...observables) : of(true);
   }
 
+  private blobifySounds(httpClient: HttpClient, progress: ProgressService): Observable<boolean> {
+
+    const observables = [];
+
+    this.boards.forEach((board: OBFBoard) => {
+      observables.push(this.progressObservable('Pre-caching sounds.', progress));
```

Comment:
Should this be outside of the forEach loop?

Reply:
or do we want to add the board id to the message and keep it inside the loop?

Reply:
I think it should be outside... unless we get feedback that they're too slow and need more granularity?